custom/icds_reports/utils/fix_past_data_due_to_change_in_height_eligibilty.sql

```diff
@@ -0,0 +1,358 @@
+  coalesce_disabled, 
+  coalesce_minority, 
+  coalesce_resident 
+ORDER BY 
+  chm.awc_id;
+-- cost 
+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+ -- Sort  (cost=0.00..0.00 rows=0 width=0)
+ --   Sort Key: remote_scan.awc_id
+ --   ->  HashAggregate  (cost=0.00..0.00 rows=0 width=0)
+ --         Group Key: remote_scan.awc_id, remote_scan.month, remote_scan.sex, remote_scan.age_tranche, remote_scan.caste, remote_scan.coalesce_disabled, remote_scan.coalesce_minority, remote_scan.coalesce_resident
+ --         ->  Custom Scan (Citus Real-Time)  (cost=0.00..0.00 rows=0 width=0)
+ --               Task Count: 64
+ --               Tasks Shown: One of 64
+ --               ->  Task
+ --                     Node: host=100.71.184.232 port=6432 dbname=icds_ucr
+ --                     ->  Finalize GroupAggregate  (cost=148618.89..160818.19 rows=15011 width=205)
+ --                           Group Key: chm.awc_id, chm.month, chm.sex, chm.age_tranche, chm.caste, (COALESCE(chm.disabled, 'no'::text)), (COALESCE(chm.minority, 'no'::text)), (COALESCE(chm.resident, 'no'::text))
+ --                           ->  Gather Merge  (cost=148618.89..158266.32 rows=60044 width=205)
+ --                                 Workers Planned: 4
+ --                                 ->  Partial GroupAggregate  (cost=147618.83..150114.44 rows=15011 width=205)
+ --                                       Group Key: chm.awc_id, chm.month, chm.sex, chm.age_tranche, chm.caste, (COALESCE(chm.disabled, 'no'::text)), (COALESCE(chm.minority, 'no'::text)), (COALESCE(chm.resident, 'no'::text))
+ --                                       ->  Sort  (cost=147618.83..147712.65 rows=37528 width=167)
+ --                                             Sort Key: chm.awc_id, chm.sex, chm.age_tranche, chm.caste, (COALESCE(chm.disabled, 'no'::text)), (COALESCE(chm.minority, 'no'::text)), (COALESCE(chm.resident, 'no'::text))
+ --                                             ->  Parallel Append  (cost=0.56..142963.52 rows=37528 width=167)
+ --                                                   ->  Parallel Index Scan using chm_month_supervisor_id_default_102648 on child_health_monthly_default_102648 chm  (cost=0.56..142775.88 rows=37528 width=167)
+ --                                                         Index Cond: (month = '2018-05-01'::date)
+
+
+
+UPDATE "agg_child_health_2018-05-01_5" agg_child_health
+  SET
+    wasting_moderate = ut.wasting_moderate,
+    wasting_severe = ut.wasting_severe,
+    wasting_normal = ut.wasting_normal,
+    stunting_moderate = ut.stunting_moderate,
+    stunting_severe = ut.stunting_severe,
+    stunting_normal = ut.stunting_normal,
+    height_measured_in_month = ut.height_measured_in_month,
+    weighed_and_height_measured_in_month = ut.weighed_and_height_measured_in_month
+  FROM (
```

Comment:
I think this can be written far more clear as `FROM temp_agg_child_my AS ut`. The subselect is not necessary